rfcs/0145-doc-strings.md

```diff
@@ -0,0 +1,461 @@
+---
+feature: doc-strings
+start-date: 2023-03-27
+author: hsjobeki
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Standard for Docstring format
+
+Wikipedia states:
+
+> In programming, a docstring is a string literal specified in source code that is used, like a comment, to document a specific segment of code. Unlike conventional source code comments, or even specifically formatted comments like docblocks, docstrings are not stripped from the source tree when it is parsed and are retained throughout the runtime of the program. This allows the programmer to inspect these comments at run time, for instance as an interactive help system, or as metadata.
+
+# Motivation
+[motivation]: #motivation
```

Comment:
A convincing `motivations` section (to me, at least :) ) looks like:

- start by describing the current state of the things
- pinpoint a specific drawback of it
- draws a small explanation why the drawback above is bad
- proposes a fix/improvement